appveyor.yml

```diff
@@ -0,0 +1,133 @@
+#---------------------------------#
+#      general configuration      #
+#---------------------------------#
+
+# version format
+version: 0.53.0.0.{build}
```

Comment:
This looks like a Gaffer version, rather than a Cortex one, and the same goes for the VERSION environment variable below. I'm also wondering if we can do something to derive them automatically from the contents of the SConstruct file, because I fear they'll quickly get out of sync otherwise.

Reply:
Ah yes, good catch. I'll see if I can modify the build number from a script in the build process.